README.md

```diff
@@ -12,29 +12,6 @@
 [![Discord chat][discord-badge]][discord-url]
 [![Twitter][twitter-badge]][twitter-url]
 
-[actions-badge]: https://img.shields.io/github/actions/workflow/status/risc0/risc0/main.yml?branch=main
```

Comment:
kinda odd to separate these links from the ones on lines 9-13

Reply:
I agree, but that's the default behavior that remark recommends of just pushing all links at the bottom 🤷

Reply:
if we want, we could leverage https://github.com/remarkjs/remark-reference-links which would turn *all* links in reference links, thus making it more consistent. But.... ehh I'm not sure

Reply:
if we want, we could leverage https://github.com/remarkjs/remark-reference-links or remark-defsplit which would turn all links in reference links, thus making it more consistent. But.... ehh I'm not sure

Reply:
can we move the ones on lines 9-13 to the bottom to keep them together?

Reply:
not sure what you mean, the ones on lines 9-13 are badges, not references
<img width="528" alt="image" src="https://github.com/user-attachments/assets/30f317bb-2e9c-4fd3-9229-3b95c4cafcdd">

Reply:
oooh ok then my suggestion doesn't make sense :)